src/index.html

```diff
@@ -1,13 +1,540 @@
+                  >
+                    Services
+                  </a>
+                </li>
+                <li class="menu__item">
+                  <a
+                    href="#testimonials"
+                    class="menu__link"
+                  >
+                    Testimonials
+                  </a>
+                </li>
+                <li class="menu__item">
+                  <a
+                    href="#contact-us"
+                    class="menu__link"
+                  >
+                    Contact Us
+                  </a>
+                </li>
+              </ul>
+            </nav>
+
+            <a
+              href="#contact-us"
+              class="header__link"
+            >
+              Hire us
+            </a>
+          </div>
+
+          <h1 class="header__title">
+            Strategic Agency
+          </h1>
+
+          <p class="header__paragraph">
+            We believe in the power of bold ideas that can solve business challenges.
+          </p>
+
+          <div>
+            <a href="#work" class="button">
+              Learn more
+            </a>
+          </div>
+        </div>
+
+        <div class="slider">
+          <div class="slider__container">
+            <div class="slider__arrows">
+              <div>
+                <a href="#" class="slider__arrows--left"></a>
+              </div>
+              <div>
+                <a href="#" class="slider__arrows--right"></a>
+              </div>
+            </div>
+            <div class="slider__info">
+              <h4 class="slider__info--name">INtro</h4>
+              <p class="slider__info--paragraph">
+                By the same illusion which lifts the horizon.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__burger-menu burger-menu" id="burger-menu">
+      <div class="burger-menu__container">
+        <div class="burger-menu__content">
+          <div class="burger-menu__top top-actions">
+            <a
+              href="#"
+              class="burger__cross"
+            >+</a>
+          </div>
+
+          <ul class="burger-menu__list">
+            <li class="burger-menu__item">
+              <a href="#about-us" class="burger-menu__link">About us</a>
+            </li>
+
+            <li class="burger-menu__item">
+              <a href="#services" class="burger-menu__link">Services</a>
+            </li>
+
+            <li class="burger-menu__item">
+              <a href="#testimonials" class="burger-menu__link">Testimonials</a>
+            </li>
+
+            <li class="burger-menu__item">
+              <a href="#contact-us" class="burger-menu__link">Contact us</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <main class="main">
+      <section class="page__section" id="about-us">
+        <div class="whoWeAre">
+          <h3 class="whoWeAre__title">
```

Comment:
you`ve used kebab-case in previous classnames. try to avoid using different notations